### DISCUSSION TITLE
Is there a procedure I can follow to remove a car from teslamate?

### DISCUSSION THREAD

**Discussion by boneyard123** [teslamate-org/teslamate#2180] [Q&A]
I replaced a vehicle and would like to remove the data of the first vehicle and have teslamate default to the new tesla.

thanks,
Bones

**Comment by miles267** [comment]
> I replaced a vehicle and would like to remove the data of the first vehicle and have teslamate default to the new tesla.
> 
> thanks, Bones

Looking to do the same thing.  Can't find instructions yet either.

**Comment by metaben** [comment]
Did you try https://github.com/adriankumpf/teslamate/issues/706

**Reply by miles267** [reply]
Yes, unfortunately I ultimately had to delete all of my docker volumes and recreate them.  Then logged in with new Tesla tokens.  I could not restore my previous backup as when I did, Teslamate did not refresh any data.
